src/Search/Api/Query/RamlRequirement.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace eLife\Search\Api\Query;
+
+trait RamlRequirement
+{
+    public static $root = __DIR__.'/../../../../vendor/elife/api/dist/samples/';
```

Comment:
Yeh that was the aim was for breaking changes to cause these particular tests to fail. It should hopefully help with syncing prior to version 1.0.